services/docker-compose.yml

```diff
@@ -699,6 +699,8 @@ services:
       WEBSERVER_LOG_FORMAT_LOCAL_DEV_ENABLED: ${LOG_FORMAT_LOCAL_DEV_ENABLED}
       WEBSERVER_LOG_FILTER_MAPPING: ${LOG_FILTER_MAPPING}
 
+      LONG_RUNNING_TASKS_NAMESPACE_SUFFIX: webserver
```

Comment:
not sure you need all these changes. if you were using the APP_NAME for the namespace it would be already included.